app/Http/Controllers/Frontend/StatisticController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Controllers\Frontend;
+
+use App\Http\Controllers\Controller;
+use Carbon\Carbon;
+use Illuminate\Contracts\Support\Renderable;
+use Illuminate\Http\Request;
+
+class StatisticController extends Controller
+{
+    public function renderMainStats(Request $request): Renderable {
+
+        $validated = $request->validate([
+                                            'from' => ['nullable', 'date'],
+                                            'to'   => ['nullable', 'date', 'after_or_equal:from']
+                                        ]);
+
+        $from = isset($validated['from']) ? Carbon::parse($validated['from']) : Carbon::now()->subWeeks(4);
+        $to   = isset($validated['to']) ? Carbon::parse($validated['to']) : Carbon::now();
+
+        $globalStats = \App\Http\Controllers\Backend\StatisticController::getGlobalCheckInStats();
```

Comment:
Ist das der Way to go? Wie machen wir das denn bei den anderen Controllern? Das muss doch hübscher gehen, z.B. mit einem `use Foo as Bar;`-Import.

Reply:
Normal includen wir alles oben. Ist auch ein Codesmell, der sowohl von phpCS und SonarLint angekreidet wird.

Reply:
Die Klasse aus Line 22 hab ich jetzt importiert. In Verbindung mit dem nächsten Kommentar meint @jeyemwey aber vermutlich das mit `$from` und `$to`? Wie darf ich das sonst machen? Ich hätte da jetzt keine Idee.